packages/elements/src/pagination/__test__/pagination.test.js

```diff
@@ -397,56 +388,35 @@ describe('pagination/Pagination', () => {
       expect(el.page).to.equal('7');
       expect(detail.value).to.equal('7');
     });
-    it('Should not fire page-changed event when last button is click but the page is already at the maximum page', () => {
-      let eventNotFired = false;
-
-      el.page = '7';
-      setTimeout(() => {
-        previousButton.click();
-      });
 
-      oneEvent(el, 'page-changed')
-        .catch(() => {
-          eventNotFired = true;
-        })
-        .then(() => {
-          expect(el.page).to.equal('7');
-          expect(eventNotFired).to.equal(true);
-        });
-    });
-    it('Should not fire page-changed event when page changes through attribute', () => {
-      let eventNotFired = false;
+    it('Should not fire page-changed event when page changes through attribute', async () => {
+      let eventFired = null;
```

Comment:
An event can be either fired or not: `let eventFired = false`

Reply:
In this case, I think the `null` value might be better because it makes sure 100% of the `false` value will come from the `try-catch` only and it does not have two places to assign the `eventNotFired = false`

Reply:
There are three possible scenarios:
`try`, `catch` and test timeout. For `try` `eventFired = true`; for catch `eventFired = false` and if test times out, the generic error is thrown. If you care about double assignment, you can leave as undefined: `let eventFired;`

Reply:
Fixed, I prefer to leave as undefined.